sklearn/datasets/_base.py

```diff
@@ -1190,7 +1190,7 @@ def load_boston(*, return_X_y=False):
             target = raw_df.values[1::2, 2]
 
         Alternative datasets include the California housing dataset [3]_
-        (i.e. func:`~sklearn.datasets.fetch_california_housing`) and Ames
+        (i.e. :func:`~sklearn.datasets.fetch_california_housing`) and Ames
```

Comment:
I do not understand why my change was not taken into account on the rendered HTML:

https://154577-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.datasets.load_boston.html

Reply:
We also need to update the string in inside the `deprecated` decorator:

https://github.com/scikit-learn/scikit-learn/blob/9cb4e761f530e7a0708059b1312921753d056cbe/sklearn/datasets/_base.py#L1135-L1137

Reply:
Good catch, thanks.